### ISSUE TITLE
coke-stage

### ISSUE THREAD

**Issue by zhangzihui247** [idstcv/CoKe#2]
Hi, 

Thanks for this great work and codes sharing!
Here I have a question, what is the hyper-parameter 'coke-stage' defined in parser?

regards,
zihui

**Comment by qian-qi** [comment]
Hi,

Thank you for your interest. The parameter is corresponding to a variance reduction strategy for learning with single-view augmentation and it denotes the number of epochs that are trained with one-hot labels. After that, soft labels will be adopted for rest epochs as discussed in Section 3.4. The ablation study can be found in B.2.2 in appendix.
